CI/appveyor/build.sh.inc

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Options
+MAKE_J=-j8
+GCC_VERSION=7.3-win32
```

Comment:
is `GCC_VERSION` used anywhere?
MAKE_J looks a bit big (8)
maybe make it configurable:
```
if [ -z "$MAKE_J" ] ; then
        MAKE_J=$(getconf _NPROCESSORS_ONLN)
        MAKE_J=${MAKE_J:-1}
fi

```

Reply:
I will remove GCC_VERSION since it's not used.
Thanks for the code snippet to detecting number of cores. Will add it.